3-Authorization-II/2-call-api-b2c/AppCreationScripts/sample.json

```
@@ -1,82 +1,81 @@
 {
     "Sample": {
-      "Title": "A React single-page application using MSAL React to authorize users for calling a protected web API on Azure AD B2C",
-      "Level": 200,
-      "Client": "React SPA",
-      "Service": "Node.js web API",
-      "RepositoryUrl": "ms-identity-javascript-react-tutorial",
-      "Endpoint": "AAD v2.0",
-      "Provider": "B2C"
+        "Author": "Salman",
```

```suggestion
        "Author": "Salman",
```

should be your github alias (or microsoft alias?.. no no github)

---

this is not resolved